src/hooks/api/functions/loader.ts

```
@@ -1,19 +1,20 @@
 import { join, extname, dirname } from 'path';
 import { readdir } from 'fs/promises';
-import { logger } from './client.mts';
+import logger from '../../../utils/logger.ts';
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
-export async function loadPlugins(): Promise<void> {
-  const pluginsDir = join(__dirname, '../commands');
+export async function syncPlugins(plugin: string, extensions?: string): Promise<void> {
+  if (!extensions) extensions = '.ts';
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)